src/main/java/org/terasology/launcher/repositories/JenkinsRepositoryAdapter.java

```diff
@@ -22,7 +21,7 @@
 import java.util.stream.Collectors;
 
 /**
- * Repository adapter for http://jenkins.terasology.io, replacing the {@link LegacyJenkinsRepositoryAdapter}.
+ * Repository adapter for http://jenkins.terasology.io.
```

Comment:
```suggestion
 * Repository adapter for https://jenkins.terasology.io.
```

Reply:
Technically this adapter still connects to HTTP (I did not change this here, because it does not solve the problem). Should probably do that in another PR

https://github.com/MovingBlocks/TerasologyLauncher/pull/616/files/5f62c1782a256a55b594a9de9c2d829bf470733a#diff-0aa9f2a209932ac1aa89399ead549d10d3cea93f42394eafb59542b2f409ed93R36

Reply:
okay, fair enough